servicecontrol/servicecontrol-in-practice.md

```diff
@@ -1,35 +1,51 @@
-- [Heartbeats](/servicepulse/intro-endpoints-heartbeats.md) and [Custom Checks](/servicepulse/intro-endpoints-custom-checks.md) can enhance a system's monitoring capability, but they add extra noise. Often, not all endpoints are mission critical. Therefore it is worth making sure that the risks to the system if an endpoint is not available are examined and understood. All communication from endpoints with ServiceControl is performed via messaging. Adding a message to the queue every second may have little impact on when the notification shows up in ServicePulse. If ServiceControl is shutdown without pausing system Endpoints, heartbeats will continue to be added to the queue. ServiceControl will attempt to process all the messages in the queue when it restarts.
+Each capability and plugin needs to be considered for each environment to determine if the value that it provides in that environment outweighs the costs that it imposes. 
 
-##### Less can be more
+Here are some important considerations when installing and deploying ServiceControl for better performance.
 
-Moving forward, take a thoughtful approach in the adoption of plugins and features:
 
-- Read the [Capacity Planning Guide](/servicecontrol/capacity-and-planning.md) for ServiceControl.
-- Read the [Troubleshooting Guide](/servicecontrol/troubleshooting.md) before deployment in production.
-- [Turn off auditing](/nservicebus/operations/auditing.md) on all endpoints as well as [heartbeats and custom checks](/servicepulse/how-to-configure-endpoints-for-monitoring.md).
-- Perform load tests to baseline the solution.
-- When comfortable with the performance of the system try adding [Heartbeats](/servicepulse/intro-endpoints-heartbeats.md) to monitor the system again.
-- Try increasing the [heartbeat interval](/servicecontrol/plugins/heartbeat.md). Ideally heartbeat updates should not occur more frequently than ServiceControl can process them or more than Operations staff are prepared to monitor them with ServicePulse.
-- With each additional change perform a load test again adjusting the heartbeat interval satisfied with the result.
-- Repeat the process of considering the business relevance, system impact and load testing with each [Custom Check](/servicecontrol/plugins/custom-checks.md).
-- Repeat the process of considering the business relevance, system impact and load testing with each Endpoint [Audit](/nservicebus/operations/auditing.md).
+## Hardware & Installation Considerations:
 
-##### Tips and Tricks
+- Read the [Capacity Planning](/servicecontrol/capacity-and-planning.md) and [Troubleshooting](/servicecontrol/troubleshooting.md) guides for ServiceControl.
+- Each enviornment should have a dedicated ServiceControl instance.
+- In production, ServiceControl should run on a dedicated machine.
```

Comment:
I wonder if we should either copy or provide a link to https://ravendb.net/docs/article-page/2.5/csharp/server/deployment/recommended-configuration, we may as well be prescriptive about hardware specs, we know SC does not work otherwise